python-sdk/src/langwatch/telemetry/tracing.py

```
@@ -95,7 +95,12 @@ def __init__(
         skip_root_span: bool = False,
         tracer_provider: Optional[TracerProvider] = None,
     ):
-        ensure_setup(api_key=api_key)
+        ensure_setup()
+        if api_key is not None:
+            warn(
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)